Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build /go/bin/ipmi_exporter
-FROM quay.io/prometheus/golang-builder:1.15-base AS builder
```

Comment:
Please keep the `1.15-base` tag. It's better to have explicit versions here so that if someone wants to build an older release at some point they will still use the same version that was used at the time of the release.

Reply:
Noted, I changed tag.